modules/Makefile.kubernetes

```diff
@@ -85,13 +85,34 @@ kubernetes\:replace-controller:
 kubernetes\:replace:
 	@$(SELF) kubernetes:replace-service kubernetes:replace-controller
 
+# (private) Create a new deployment
+kubernetes\:create-deployment:
+	$(call kubectl_create,deployment)
+
+# (private) Delete a deploymentt
+kubernetes\:delete-deployment:
+	$(call kubectl_delete,deployment)
+
+# (private) Update existing controller using rolling-update or create a new controller; do not notify datadog
+kubernetes\:deploy-deployment:
```

Comment:
Keep it as you have, ie `deploy-deployment`. As you say it does work because `deployment` is a type here